gradio-blocks/README.md

```diff
@@ -0,0 +1,114 @@
+- [https://huggingface.co/spaces/multimodalart/mindseye-lite](https://huggingface.co/spaces/multimodalart/mindseye-lite)
+- [https://huggingface.co/spaces/akhaliq/ArcaneGAN-blocks](https://huggingface.co/spaces/akhaliq/ArcaneGAN-blocks)
+- [https://huggingface.co/spaces/merve/gr-blocks](https://huggingface.co/spaces/merve/gr-blocks)
+- [https://huggingface.co/spaces/osanseviero/tortoisse-tts](https://huggingface.co/spaces/osanseviero/tortoisse-tts)
+- [https://huggingface.co/spaces/akhaliq/CaptchaCracker](https://huggingface.co/spaces/akhaliq/CaptchaCracker)
+
+
+To participate in the event
+
+- Join the organization for Blocks event
+    - [https://huggingface.co/Gradio-Blocks](https://huggingface.co/Gradio-Blocks)
+- Join the discord
+    - [discord](https://discord.com/invite/feTf9x3ZSB)
+
+
+Participants will be building and sharing Gradio demos using the Blocks feature. We will share a list of ideas of spaces that can be created using blocks or participants are free to try out their own ideas. At the end of the event, spaces will be evaluated for creativity and prizes will be given. 
+
+
+potential ideas for creating spaces:
+
+
+- papers from https://paperswithcode.com/
+- themed spaces: Vision, NLP, Audio based spaces
+- Models from huggingface model hub
+- Models from other model hubs
+    - Tensorflow Hub: see example Gradio demos at https://huggingface.co/tensorflow
+    - Pytorch Hub: see example Gradio demos at https://huggingface.co/pytorch
+    - ONNX model Hub: see example Gradio demos at https://huggingface.co/onnx
+    - PaddlePaddle Model Hub: see example Gradio demos at https://huggingface.co/PaddlePaddle
+- participant ideas, try out your own ideas
+
+
+**Prizes**
+
+- [Hugging Face PRO subscription](https://huggingface.co/pricing) 1 month for top 25 winners and 1 year for 1st place winner!
+- [Hugging Face PRO subscription](https://huggingface.co/pricing) badge on Hugging Face for winners
+- Gradio badge for everyone participating!
+- Blocks event badge on HF for all participants!
+- Swag from [Hugging Face merch shop](https://huggingface.myshopify.com/): t-shirts, hoodies, mugs of your choice for top 10 winners!
+- Embedding your Gradio Blocks demo in the Gradio Blog for 1st place winner
+- Gradio Team Office Hour for 1st place winner
+
+**Prizes Criteria**
+
+- Staff Picks
+- Most liked Spaces
+- Community Pick (voting)
+- Most Creative Space (voting)
+
+
+**Creating a Gradio demo on Hugging Face Spaces**
+
+Once a model has been picked from the choices above or feel free to try your own idea, you can share a model in a Space using Gradio
+
+Read more about how to add Gradio spaces [here](https://huggingface.co/blog/gradio-spaces).
+ 
+Steps to add Gradio Spaces to the Gradio Blocks Party org
+1. Create an account on Hugging Face
+2. Join the Gradio Blocks Party Organization by clicking "Join Organization" button in the organization page
```

Comment:
Could link https://huggingface.co/Gradio-Blocks here again, otherwise readers will have to look for it higher up in the blog post.

Reply:
will link thanks

Reply:
updated